STYLE.md

```diff
@@ -0,0 +1,52 @@
+# Distributed Tracing Working Group Style Guide
+
+This document describes the editorial style preferred by the W3C Distributed Tracing Working Group.
+Where possible and appropriate, the rules in this document should be followed when making pull requests to repositories maintained by the working group.
+If a rule is not appropriate or possible in context, it may be acceptable to break the rule with the consent of the working group members reviewing a pull request.
+If it is not obvious why a rule is broken, a comment should be made in the document describing the reason.
+
+# Markdown
+
+The following section applies to all markdown files.
+
+## File extension
+
+All markdown files should have the extension `.md`.
+
+## Line Breaks
+
+Each sentence should be on its own line.
+This allows for a pull request review to make a suggestion to an individual sentence,
+which is a natural unit of language, without changing the line breaks for unrelated lines.
+If a line is particularly long, it may also be broken on other punctuation such as commas or semicolons.
+
+```markdown
+This is a full sentence.
+This is another full sentence, which is longer but still continues to be on the same line.
+This is yet another full sentence, which is quite a bit longer than previous sentences,
+and has a line break after a comma in order to keep line lengths reasonable.
+```
+
+## Referencing external RFCs
+
+I personally like the regular markdown link syntax. The linked RFC is still included in the bibliography and it reads more naturally to me. It is much easier to link [RFC0000 Section x.y](https://link.to.rfc) than to read a full URL and linking only to the bibliography is annoying for the user who clicks the link to go to the section referenced.
```

Comment:
👍  I also prefer that style. We should probably remove this line ("I personally like") from the PR though, right?

Reply:
lol oops. Copied from the original issue.

Reply:
I copied the whole issue over and formalized it. I guess I just forgot to delete that paragraph